NuimoSDK/NuimoGestureEvent.cs

```diff
@@ -21,10 +21,30 @@ public enum NuimoGesture
         SwipeRight,
         SwipeUp,
         SwipeDown,
+        TouchLeft,
+        TouchRight,
+        //TouchTop seems not to work
+        TouchTop,
+        TouchBottom,
+        LongTouchLeft,
+        LongTouchRight,
+        //LongTouchTop seems not to work
+        LongTouchTop,
+        LongTouchBottom,
         FlyLeft,
         FlyRight,
+        //Removed from firmware?
         FlyBackwards,
+        //Removed from firmware?
         FlyTowards,
-        FlyUpDown,
+        FlyUpDown
+    }
+
+    public enum NuimoColor
+    {
+        Unknown,
+        Black,
+        //TODO White or Silver?
+        Silver
```

Comment:
Renamed to White, also added Gold from https://github.com/AravinthPanch/nuimo-bluetooth-low-energy-python